### ISSUE TITLE
Improve Issue Template

### ISSUE THREAD

**Issue by jens1o** [vscode-icons/vscode-icons#936]
**Checklist:**

- [X] I'm sure this issue is not a *duplicate*?

I think we should improve the issue template, so when someone's wanting to create a folder request, they get notified that they should do it here: #650 

Thoughts?

**Comment by robertohuertasm** [comment]
See my comment [here](https://github.com/vscode-icons/vscode-icons/issues/935#issuecomment-303000554).

**Comment by JimiC** [comment]
I agree with @jens1o.

**Comment by robertohuertasm** [comment]
But why to treat folder icons in a different way? Is it not better to have an issue per request so we can have a clean discussion about each request? An even when submitting a PR the reference is cleaner.

I think grouping the folder icons makes sense if there are several icons that have similar characteristics (images and icons and so on) and that's why I thought it was a good idea. But, when there's a unique demand like in #935 I honestly don't see the point. By having a specific issue, we can close the discussion and reference the specific issue in case someone else asks for the same and not clutter the multiple folder issue, which would be the case if we start discussing about the convenience of each of the icons.

Basically, my point is that we should treat folder icons the same way we treat file icons. IMHO it would be better for users and for us to manage each icon request.

**Comment by JimiC** [comment]
@robertohuertasm If it were up to me I would apply the same for the file icons as well. Currently, we have 30+ issues open with file icon requests simple because we either haven't found a suitable icon or we lack the time to do the artwork. Imagine what will happen if we add the 40+ requests for folder icons. Personally, I would spend more time tracking down one by one the requests than having them all on one issue at a glance. It's purely a perspective of time management.

I admit that the folder issue is getting cluttered but this is happening solely to the reason that we don't submit any artwork for them to remove entries from it.

Personally, I'm facing a family issue that takes up all my free time. I expect things to smooth out on July.

**Comment by robertohuertasm** [comment]
I see your point, but I personally prefer having many separate issues with their own thread than one huge issue with lots of threads inside. You said is for the sake of time management. I think it's way more easy to tackle the icons one PR after the other and reference and close its specific related issue. The same goes for discussion tracking.

Anyway, if you both agree with this I'm also in 😉 👍 

@JimiC hope your family issue solves in a good way and soon 😃

**Comment by JimiC** [comment]
What's the decision on this? Users keep opening new requesting issues, instead of adding to #650, for icons already mentioned there.

**Comment by robertohuertasm** [comment]
I thought we agreed in modifying the template to let users know about it.

**Comment by JimiC** [comment]
Yeah, but nobody did it. PR submitting pending.

**Comment by robertohuertasm** [comment]
Cool. I thought that @jens1o was on it as he was the one who opened the issue. :stuck_out_tongue_closed_eyes:
